services/mailer/mail.go

```diff
@@ -68,6 +69,7 @@ func SendUserMail(language string, u *models.User, tpl base.TplName, code, subje
 
 	var content bytes.Buffer
 
+	// TODO: i18n
```

Comment:
if you add the locale as "i18n" into the data map above then making things translatable is free